### ISSUE TITLE
Allow to use "emptyLineBefore" with "unspecified"

### ISSUE THREAD

**Issue by evgenymarkov** [hudochenkov/stylelint-order#77]
Hi, I would very much like to have possibility to put together unspecified properties into a block of code separated by empty line.

Example:
![screenshot 2018-12-15 at 18 18 29](https://user-images.githubusercontent.com/16443248/50043450-32ddc580-0096-11e9-906f-02ac39608310.png)

Now it’s impossible to do so. It would be good if it was achieved by the following settings:

```javascript
    "order/properties-order": [
      [
        { /* ... */ },
        { /* ... */ },
        { /* ... */ }
      ],
      { 
        "unspecified": "bottomAlphabetical",
        "emptyLineBefore": "always"
      }
    ]
  }
```

**Comment by hudochenkov** [comment]
It's a good idea!

I would name the new secondary option `unspecifiedEmptyLineBefore`. Looks a little bit weird, but it shows what empty line for. Possible values: `"always"`, `"never"`. If an option is not specified, ignore it.

**Comment by hudochenkov** [comment]
Released in `3.0.0`.
